nss-day-cohort-46/enthusiastic-enunons-client
# Kb view tags client

# Description
Added create tag form and tag list

## Type of change



# Testing Instructions

you should be able to create a new tag
```
git fetch —all
git checkout kb-viewTagsClient

```

# Checklist:
